internal/api/api.go

```diff
@@ -46,19 +46,22 @@ func NewAPI(cfg Config, serviceManager *servicemanager.Client, secretProvider *c
 }
 
 func (a *API) Start() {
-	mux := http.NewServeMux()
-	mux.HandleFunc("GET /api/secrets", a.ListSecrets)
-	mux.HandleFunc("GET /api/service-instances", a.ListServiceInstances)
-	mux.HandleFunc("PUT /api/service-instance/{id}", a.CreateServiceInstance)
-	mux.HandleFunc("GET /api/service-instance/{id}", a.GetServiceInstance)
-	mux.HandleFunc("GET /api/service-offerings/{namespace}/{name}", a.ListServiceOfferings)
-	mux.HandleFunc("GET /api/service-offering/{id}", a.GetServiceOffering)
-	mux.Handle("GET /", http.FileServer(a.frontendFS))
-	a.server.Handler = mux
-
+	router := http.NewServeMux()
+	a.AttachRoutes(router)
+	a.server.Handler = router
 	log.Fatal(a.server.ListenAndServe())
 }
 
+func (a *API) AttachRoutes(router *http.ServeMux) {
+	router.HandleFunc("GET /api/secrets", a.ListSecrets)
+	router.HandleFunc("GET /api/service-instances", a.ListServiceInstances)
```

Comment:
Unlike for service-offerings we do not allow to specify namespace and name. Why is that? Should it be consistent or not.

Reply:
I am not changing the path in this PR.